testpilot/frontend/static-src/styles/modules/_experiment-details.scss

```diff
@@ -1,5 +1,18 @@
-.details-header-wrapper {
+.is-enabled {
```

Comment:
let's stick with the flex-box convention we're doing elsewhere here...

nix the `line-height` and `text-align` rules
and add the following rules:

```
@include flex-container(row, center, center);
height: 36px;
padding-top: 1px;